gitlab/utils.py

```diff
@@ -161,3 +161,28 @@ def warn(
         stacklevel=stacklevel,
         source=source,
     )
+
+
+def _validate_attrs(
+    data: Dict[str, Any],
+    attributes: types.RequiredOptional,
+    excludes: Optional[list] = None,
```

Comment:
I'm pretty sure it can only be a string.

Will need to import `List` on the `from typing ...` above around line 22

```suggestion
    excludes: Optional[List[str]] = None,
```

Reply:
Question. I notice `excludes` is only called in one place and it can only pass in a `list` containing one string. Is there a reason to use a `list` if it only has one item?

Reply:
I don't have an answer for that either. @nejch can better answer but I suspect you are right.

Reply:
done

Reply:
> done

What is "done"?